server/src/main/java/com/lunchtime/controllers/PersonController.java

```diff
@@ -20,33 +18,27 @@
 
     private final PersonService personService;
     private final JwtUtil jwtUtil;
-    private final AuthController authController;
-    private final TokenHistory tokenHistory;
 
     @PostMapping
     public ResponseEntity<?> createPerson(
         @Valid @RequestBody RegisterPerson registerPerson) throws Exception {
 
         if (registerPerson.getId() != null) {
-            return ResponseEntity.badRequest()
-                .build();
+            return null;
         }
 
         PersonDto personDto = null;
         try {
             personDto = personService.saveRegisterPerson(registerPerson);
         } catch (NonUniqueResultException nue) {
-            return ResponseEntity.status(Integer.parseInt(nue.getMessage())).build();
+            return ResponseEntity.status(409)
+                .body(nue.getMessage());
         } catch (Exception e) {
             e.printStackTrace();
         }
-        
+
         if (personDto != null) {
-            JwtPersonDetails jwtPersonDetails = new JwtPersonDetails(
-                registerPerson.getEmail(), registerPerson.getPassword());
-            authController.createAuthenticationToken(jwtPersonDetails);
-            String token = tokenHistory.getTokenList().remove(0);
-            return ResponseEntity.ok(token);
+            return ResponseEntity.ok(personDto);
```

Comment:
Your task is - when user tries to register account with password that equals phone number or email, he sees warning

Reply:
And what is violated in Controller?